pyproject.toml

```diff
@@ -69,7 +69,8 @@ all = [
     "apache-airflow-providers-snowflake",
     "snowflake-sqlalchemy>=1.2.0,<=1.2.4",
     "snowflake-connector-python[pandas]",
-    "apache-airflow-providers-postgres"
+    "apache-airflow-providers-postgres",
+    "sphinx==4.4.0"
```

Comment:
We should move this to a separate extra - `dev` or `doc` as users don't need to install sphinx

Reply:
Yes @kaxil totally agree, have create a separate extra - 'doc'.